src/main/java/frc/robot/subsystems/intake/intakeConstants.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+public interface intakeConstants {
+    int MOTOR_ID = 0; //TODO: find id
+    int LIMITSWITCH_ID = 0; //TODO: find id
+
+    //motor constants
+    double MOTOR_CURRENT=0.5; //TODO: find CURRENT
+    double FEED_SPEED=0.3; //TODO: find SPEED
```

Comment:
You dont need 2

Reply:
they are for differant steps
do i need to use only 1 constant?